packages/modeling/src/operations/extrusions/slice/repairSlice.js

```diff
@@ -5,42 +5,54 @@ const vec3 = require('../../../maths/vec3')
  */
 const repairSlice = (slice) => {
```

Comment:
Why not just repair? If used with full name spacing then this would become...

`
extrutions.slice.repairSlice(myslice)
`

Looks a bit strange.

Also, slices should be immutable like other geometry. Is repair() being used somewhere?

Reply:
The repair slice name existed before this PR. But, I just pushed changes:

- rename `repairSlice` to `repair`
- made repair immutable

Repair is being used by all the extrude functions (linear, fromSlices, etc).